_includes/footer.html

```diff
@@ -4,7 +4,7 @@
       <div class="col-lg-9 col-xs-5">
         <nav class="footer-nav">
           <a href="/events/">Events</a>
-          <a href="/coc/">Code of Conduct</a> 
+          <a href="/conduct/">Code of Conduct</a> 
```

Comment:
setup a redirect in jekyll:

https://help.github.com/articles/redirects-on-github-pages/